eng/illink.targets

```diff
@@ -192,6 +192,13 @@
       <ILLinkArgs>$(ILLinkArgs) --disable-opt unusedinterfaces</ILLinkArgs>
       <!-- keep DynamicDependencyAttribute unless a project explicitly disables it -->
       <ILLinkArgs Condition="'$(ILLinkKeepDepAttributes)' != 'false'">$(ILLinkArgs) --keep-dep-attributes true</ILLinkArgs>
+      <!-- enable verbose output -->
+      <ILLinkArgs>$(ILLinkArgs) --verbose</ILLinkArgs>
```

Comment:
In case these ever need to be conditioned out for a specific reason, can we add a property that individiual projects can override for this?

Reply:
I believe this is going to be removed as soon as the linker enables warnings by default. So there is probably no reason to add a property here - it is just going to be deleted.